Source/DafnyPipeline.Test/Boogie.cs

```diff
@@ -0,0 +1,219 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using DiffPlex.DiffBuilder;
+using DiffPlex.DiffBuilder.Model;
+using Microsoft.VisualStudio.TestPlatform.PlatformAbstractions;
+using Xunit;
+
+namespace DafnyPipeline.Test
+{
+    public class Boogie
```

Comment:
nit: consider naming it this "Translator" or something similar, since this isn't a test of Boogie itself.